lib/rspec/matchers/hash_formatting.rb

```diff
@@ -0,0 +1,20 @@
+module RSpec
+  module Matchers
+    # @api private
```

Comment:
If these are still to be private they probably could have been left in `Pretty`...